MultiFQtoVC.nf

```diff
@@ -684,18 +689,114 @@ intervals = Channel
 gI = intervals
   .map {a -> [a,a.replaceFirst(/\:/,"_")]}
 
+muTect1Intervals = Channel.create()
 muTect2Intervals = Channel.create()
 varDictIntervals = Channel.create()
 strelkaIntervals = Channel.create()
 
 Channel
   .from gI
-  .separate (muTect2Intervals, varDictIntervals, strelkaIntervals) {a -> [a, a, a]}
+  .separate (muTect1Intervals, muTect2Intervals, varDictIntervals, strelkaIntervals) {a -> [a, a, a, a]}
 
 // now add genomic intervals to the sample information
 // join [idPatientNormal, idSampleNormal, bamNormal, baiNormal, idSampleTumor, bamTumor, baiTumor] and ["1:1-2000","1_1-2000"] 
 // and make a line for each interval
 
+if ('MuTect1' in workflowSteps) {
+  bamsFMT1 = bamsForMuTect1.spread(muTect1Intervals)
+  bamsFMT1 = logChannelContent("Bams for MuTect1: ", bamsFMT1)
+
+  process RunMutect1 {
+    publishDir "VariantCalling/MuTect1/intervals"
+
+    module 'bioinfo-tools'
+    module 'java/sun_jdk1.7.0_25'
+    module 'mutect/1.1.5'
+
+    cpus 8 
+    memory { 16.GB * task.attempt }
+    time { 16.h * task.attempt }
+    errorStrategy { task.exitStatus == 143 ? 'retry' : 'terminate' }
+    maxRetries 3
+    maxErrors '-1'
+
+    input:
+    set idPatient, idSampleNormal, file(bamNormal), file(baiNormal), idSampleTumor, file(bamTumor), file(baiTumor), genInt, gen_int from bamsFMT1
+
+    output:
+    set idPatient, idSampleNormal, idSampleTumor, val("${gen_int}_${idSampleNormal}_${idSampleTumor}"), file("${gen_int}_${idSampleNormal}_${idSampleTumor}.mutect1.vcf") into mutect1VariantCallingOutput
+  
+    """
+    java -Xmx${task.memory.toGiga()}g -jar \${MUTECT_HOME}/muTect.jar \
+    -T MuTect \
+    -R ${refs["genomeFile"]} \
+    --cosmic ${refs["cosmic41"]} \
+    --dbsnp ${refs["dbsnp"]} \
+    -I:normal $bamNormal \
+    -I:tumor $bamTumor \
+    -L \"${genInt}\" \
+    --out ${gen_int}_${idSampleNormal}_${idSampleTumor}.mutect1.txt \
```

Comment:
The main output of mutect is usually called "call_stats.out", might be good to make this .mutect1.call_stats.out rather than mutect1.txt for clarity.